internal/config/profile.go

```diff
@@ -534,7 +534,7 @@ func (p *Profile) Delete() error {
 }
 
 func (p *Profile) Filename() string {
-	return filepath.Join(p.configDir, ToolName+".toml")
+	return filepath.Join(p.configDir, "config.toml")
```

Comment:
FYI: we were storing the atlasCLI config in the path `.config/atlascli/atlascli.toml` instead of `.config/atlascli/config.toml`. This should fix the issue 💯